packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec

```diff
@@ -5,18 +5,12 @@
 
 Name: rubygem-%{gem_name}
 Version: 12.1.0
-Release: 1%{?foremandist}%{?dist}
+Release: 2%{?foremandist}%{?dist}
 Summary: Connects Foreman with Red Hat Cloud services
 License: GPLv3
 URL: https://github.com/theforeman/foreman_rh_cloud
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-Obsoletes: %{?scl_prefix}rubygem-redhat_access
```

Comment:
Would it be safer to version the obsolete? I guess i'm unsure if we still need this or if it's an artifact we can get rid of.

Reply:
that name is so old, it can be legally drinking in the US, remove it!